input.py

```diff
@@ -0,0 +1,17 @@
+def input_values():
+    unmerged_meetings = []
+    print("Input your data:")
+    while True:
```

Comment:
Using "while True" is not a good idea because it makes it hard to maintain the code and the way you are escaping the loop is considered bad practice.
Try:
```
do
...
while ...
```
instead